upup/pkg/fi/cloudup/awsup/aws_utils.go

```diff
@@ -37,62 +40,42 @@ import (
 )
 
 // allRegions is the list of all regions; tests will set the values
-var allRegions []*ec2.Region
+var allRegions []ec2types.Region
 var allRegionsMutex sync.Mutex
 
-// isRegionCompiledInToAWSSDK checks if the specified region is in the AWS SDK
-func isRegionCompiledInToAWSSDK(region string) bool {
```

Comment:
this is no longer possible, see https://github.com/aws/aws-sdk-go-v2/discussions/2586